OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/FirebaseAnalytics.java

```diff
@@ -591,4 +591,22 @@ public void trackExperimentParams(String experimentName, Map<String, String> val
         event.putMap(values);
         logFirebaseEvent(event.getName(), event.getBundle());
     }
+
+    @Override
+    public void trackCastDeviceConnectionChanged(@NonNull String eventName, @NonNull String connectionState, @NonNull String type) {
+        final FirebaseEvent event = new FirebaseEvent(eventName);
```

Comment:
Pass bi value in the constructor to sync it with other functions.

Reply:
done.